test/helpers/constants.ts

```diff
@@ -33,23 +33,23 @@ export const RUNTIME_CONSTANTS = {
     MAX_FEE_MULTIPLIER: 100_000_000_000_000_000_000_000n,
     MIN_BASE_FEE_IN_WEI: "125000000",
     MAX_BASE_FEE_IN_WEI: "125000000000000",
-    TARGET_FILL_PERMILL: 500_000n,
+    TARGET_FILL_PERMILL: 350_000n,
```

Comment:
If you change those variables, they will break existing smoke test.
Ideally this should be changed to a different variable, or if we do it too much, to a map<version, value>